frontend/src/main/components/Nav/AppNavbar.js

```diff
@@ -97,6 +97,16 @@ export default function AppNavbar({ currentUser, systemInfo, doLogout, currentUr
                   </NavDropdown>
                 )
               }
+              {
+                hasRole(currentUser, "ROLE_MEMBER") && (
+                  <Nav.Link as={Link} to="/apply/rider">Apply to be a Rider</Nav.Link>
+                )
+              }
+              {
+                hasRole(currentUser, "ROLE_ADMIN") && (
+                  <Nav.Link as={Link} to="/admin/applications/riders">Rider Applications</Nav.Link>
```

Comment:
This page doesn't seem to have an  entry in `App.js`, so it doesn't seem to be accessible for the admins.

Is that intentional?  Did I overlook something? 

Or is this a bug that needs to be fixed?